.devcontainer/setup.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+ln -sfn /workspaces/gt4py/.devcontainer/.vscode /workspaces/gt4py/.vscode
+python -m venv .venv
+source .venv/bin/activate
+pip install --upgrade pip setuptools wheel
+pip install -e .
+pip install -r requirements-dev.txt
+pip install -i https://test.pypi.org/simple/ atlas4py
```

Comment:
If we install `uv` in the dockerfile then we can use it here which should be nicer and much faster:
```suggestion
uv venv .venv
source .venv/bin/activate
uv pip install -r requirements-dev.txt
uv pip install -e .
uv pip install -i https://test.pypi.org/simple/ atlas4py
```